client/src/pages/S3Test.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from "react";
+import S3 from "aws-s3";
+
+const config = {
+    bucketName: 'harvestcrunch-bakedgoods',
+    region: 'us-east-2',
+    accessKeyId: process.env.REACT_APP_AWS_ACCESS_KEY_ID,
+    secretAccessKey: process.env.REACT_APP_AWS_SECRET_KEY,
+}
+class S3Test extends Component{
+    state = {
+        selectedFile: null,
+    }
+    fileSelectedHandler = event => {
+        this.setState({
+            selectedFile: event.target.files[0]
+        });
+    }
+
+    fileUploadHandler = () => {
+        const S3Client = new S3(config);
+        var filename = this.state.selectedFile.name.split(".")[0];
+        S3Client
+            .uploadFile(this.state.selectedFile, filename)
+            .then(data => {
+                var link = document.getElementById("picLink");
+                link.innerHTML = data.location;
```

Comment:
save the data in a state instead of manipulating the DOM 

we want to be avoiding this because we're using React :)

Reply:
actually you ignore this because you'll probably overwrite this very soon anyways